### ISSUE TITLE
How to rename the name of a functions used for a model?

### ISSUE THREAD

**Issue by Justin-king-de** [marcoslucianops/DeepStream-Yolo#38]
Hello @marcoslucianops . There seems to be a lot of change in the repo since the last time I used it to convert my yolov4-tiny model. Last time, I used your `nvdsparsebbox_Yolo.cpp` file which you have edited to add support for yolov4-tiny about 3 months back. Now this file in your repo is completely changed though. However, my question is I want to run yolov4 tiny model in deepstream but I want its name to be changed to another name instead of yolov4.

As of now, in `nvdsparsebbox_Yolo.cpp` file, I see there is a function with name` NvDsInferParseCustomYoloV4` and other related functions to it like `convertBBoxYoloV4` `addBBoxProposalYoloV4` ` decodeYoloV4Tensor` `NvDsInferParseYoloV4`. And now in config file we are giving the name to the function `NvDsInferParseCustomYoloV4`.

If I change the names of all these functions with some other name, say model ex:`NvDsInferParseCustommodel` ,`convertBBoxmodel`,` addBBoxProposalmodel` etc and give the name of function as `NvDsInferParseCustommodel` in config file, will it work?
So basically I am replacing the name yolov4 with some other name and calling the renamed function from config file. Will this work?

**Comment by marcoslucianops** [comment]
Hi, I recommend to use the latest files from my repo for better compatibility and performance on DeepStream. You can rename these functions, it's just function names. In my latest files, I use one function (NvDsInferParseYolo) for all models, excepting YOLOv2 (it have a separated function named NvDsInferParseYoloV2).
